tests/test_routes.py

```diff
@@ -383,28 +383,28 @@ def test_add_items_with_invalid_price(self):
     def test_update_name_of_shopcart(self):
         """It should return the shopcart within updated name"""
         test_shopcart = ShopcartFactory()
-        resp = self.client.post(self.base_url, json=test_shopcart.serialize())
+        resp = self.client.post(self.base_url_restx, json=test_shopcart.serialize())
         self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
 
         new_shopcart = resp.get_json()
         new_shopcart["name"] = "DevOps"
         new_shopcart_id = new_shopcart["id"]
-        resp = self.client.put(f"{self.base_url}/{new_shopcart_id}", json=new_shopcart)
+        resp = self.client.put(f"{self.base_url_restx}/{new_shopcart_id}", json=new_shopcart)
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
         updated_shopcart = resp.get_json()
         self.assertEqual(updated_shopcart["name"], "DevOps")
 
     def test_update_non_existent_shopcart(self):
```

Comment:
this test case duplicates with `def test_update_a_nonexistent_shopcart()`, can you help remove one of them? thanks!

Reply:
done